samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -96,6 +103,41 @@ private static void AddServices(IServiceCollection services, ConfigurationManage
         services.AddSemanticKernelServices(configuration);
     }
 
+    private static void AddAuth(this IServiceCollection services, ConfigurationManager configuration)
```

Comment:
It's not just authorization - it's (at least with AAD), authentication and authorization, or AuthN and AuthZ, or just auth for short.  See comment above.

Reply:
(as above) ASP.NET's naming convention seems to use Authorization for all of this, mostly because we only care about authorized access at the endpoints of a service. Unlike auth providers (like AAD) which handle both authentication and authorization flows.